examples/06_wifi_scan/main/component.mk

```diff
@@ -0,0 +1,10 @@
+#
+# Main Makefile. This is basically the same as a component makefile.
+#
+# This Makefile should, at the very least, just include $(SDK_PATH)/make/component_common.mk. By default, 
```

Comment:
The newly updated esp-idf build system doesn't require including component_common.mk, so you can delete everything from this line down.